src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DiskStationProxyBase.cs

```diff
@@ -105,115 +179,109 @@ private void AuthenticateClient(DownloadStationSettings settings)
                  { "method", "login" },
                  { "account", settings.Username },
                  { "passwd", settings.Password },
-                 { "format", "cookie" },
+                 { "format", "sid" },
                  { "session", "DownloadStation" },
              };
 
-            var authLoginRequest = BuildRequest(settings, DiskStationApi.Auth, arguments, HttpMethod.GET);
-            authLoginRequest.StoreResponseCookie = true;
+            var authInfo = GetApiInfo(DiskStationApi.Auth, settings);
 
-            var response = _httpClient.Execute(authLoginRequest);
+            var requestBuilder = BuildRequest(settings, authInfo, HttpMethod.GET);
```

Comment:
Now you have the requestBuilder, isn't it easier to just call AddQueryParam, instead of the dictionary? This applies in multiple cases.
``` 
var requestBuilder = BuildRequest(settings, authInfo, HttpMethod.GET);
requestBuilder.AddQueryParam("version", "1");
requestBuilder.AddQueryParam("method", "login");
requestBuilder.AddQueryParam("account", settings.Username);
requestBuilder.AddQueryParam("passwd", settings.Password);
requestBuilder.AddQueryParam("format", "sid");
requestBuilder.AddQueryParam("session", "DownloadStation");
```

BuildRequest can add the 'api' queryparam, and potentially deal with version & method too via an method parameter, simply because all api calls have those query params. And gives the BuildRequest a chance to verify your specified version vs the MinMax.